CHANGELOG.md

```diff
@@ -5,7 +5,7 @@
   - [#98](https://github.com/Datatamer/unify-client-python/issues/98) Add `__geo_interface__` to Dataset
   - [#100](https://github.com/Datatamer/unify-client-python/issues/100) Add `from_geo_features` to Dataset
   - [#116](https://github.com/Datatamer/unify-client-python/issues/116) Add support for associating a dataset with a project
-
+  - [#109](https://github.com/Datatamer/unify-client-python/issues/109) Add support for profiling datasets
```

Comment:
```suggestion
  - [#109](https://github.com/Datatamer/unify-client-python/issues/109) Add support for profiling datasets
  
```

Reply:
@mollysacks ^sorry if I was unclear: there should be exactly 1 empty line between CHANGELOG sections. Previously there were 2 and it looks like you've removed both rather than keeping 1.